public/locales/en/app.ftl

```diff
@@ -0,0 +1,185 @@
+pwt-summary-5 = 2FA offers an extra layer of protection by requiring you to enter additional information (like a code sent via text) before you can access your accounts.
+pwt-summary-6 = We'll let you know if your account information is compromised in a data breach or exposed to hackers in some other way.
+
+landing-headline = Your right to be safe from hackers starts here.
+landing-blurb = {-product-name-nowrap} arms you with tools to keep your personal information safe. Find out what hackers already know about you and learn how to stay a step ahead of them.
+
+scan-label = Try it out. Enter your email for a basic scan.
+scan-placeholder = Enter Email Address
+scan-privacy = Your email will not be stored.
+scan-submit = Scan
+scan-another-email = Scan Another Email Address
+scan-featuredbreach-label = Find out if your <span class="bold"> { $featuredBreach } </span> account was compromised.
+scan-error = Must be a valid email.
+
+signup-banner-headline = Stay safe with {-product-name-nowrap} protection.
+signup-banner-blurb = Sign up for {-product-name-nowrap}. You’ll get a full report on your compromised accounts and notifications any time your accounts appear in new data breaches.
+signup-button = Sign up
+
+download-firefox-bar-blurb = {-product-name-nowrap} is brought to you by the <span class="nowrap">all-new {-brand-name}</span>.
+download-firefox-bar-link = Download {-brand-name} now
+
+download-firefox-banner-blurb = Take control of your browser
+download-firefox-banner-button = Download {-brand-name}
+
+signup-modal-headline = Sign Up for {-product-name-nowrap}
+signup-modal-blurb = Sign up for your full report, alerts when new breaches happen, and safety tips from {-product-name-nowrap}.
+signup-modal-close = Close
+
+signup-modal-verify-headline = Verify Your subscription
+signup-modal-verify-blurb = We sent a verification link to <span id="submitted-email" class="medium"></span>.
+signup-modal-verify-expiration = This link expires in 24 hours.
+signup-modal-verify-resend = Not in inbox or spam folder? Resend.
+
+signup-with-fxa - Sign Up with {-brand-name} Account
+
+form-signup-placeholder = Enter email
+form-signup-checkbox = Get the latest from {-brand-Mozilla} and {-brand-name}.
+form-signup-submit = Sign up
+form-signup-error = Must be a valid email.
+
+
+found-breaches-headline = This could be a problem...
+no-breaches-headline = So far, so good
+
+featured-breach-results = 
+  Your account appeared in the <span class="bold"> { $featuredBreach } </span> breach, 
+    { $breachCount ->
+        [0] but did not appear in any other known data breaches
+        [1] as well as { $breachCount } other breach
+       *[other] as well as { $breachCount } other breaches
+    }.
+
+scan-results = Your account { $breachCount ->
+    [0] Your email address did not appear in our basic scan.
```

Comment:
@flodolo thanks for the feedback. Is it possible to translate any variant in Pontoon, or is there a specific issue with the zero case?

Reply:
<img width="1363" alt="screen shot 2018-10-16 at 4 25 07 pm" src="https://user-images.githubusercontent.com/22355127/47048485-26092800-d160-11e8-97f6-d8c327e9f5f7.png">
Was following this example and a similar example of usage here: https://projectfluent.org/play/

Reply:
This is perfectly fine Fluent, it's just tricky for localizers to understand what we're asking them to translate.

It becomes easier if we're exposing the full phrase for each variant.

There's things we'll want to do on the l10n tooling side there in the next year, but right now, the extra repetition of the head and tail section would be good.

```properties
single-featured-breach-result = Your account appeared in the <span class="bold"> { $featuredBreach } </span> breach, but did not appear in any other known data breaches.
multiple-featured-breach-results =
  {  $breachCount ->
        [one] Your account appeared in the <span class="bold"> { $featuredBreach } </span> breach, as well as { $breachCount } other breach.
       *[other] Your account appeared in the <span class="bold"> { $featuredBreach } </span> breach, as well as { $breachCount } other breaches.
  }
```

Reply:
@Pike thanks for the clarification

Reply:
What Pike said :)

The UI looks [like this](https://pontoon.mozilla.org/it/firefox/browser/browser/preferences/preferences.ftl/?search=%5B1%5D&string=176248). The fragment before would become another input field, and you would need to drag the input field to see everything.

Reply:
To be clear, I would do this

```
featured-breach-no-results = 
  Your account appeared in the <span class="bold"> { $featuredBreach } </span> breach, 
  but did not appear in any other known data breaches.

featured-breach-results =
    { $breachCount ->
        [one] Your account appeared in the <span class="bold"> { $featuredBreach } </span> breach, as well as { $breachCount } other breach.
      *[other] Your account appeared in the <span class="bold"> { $featuredBreach } </span> breach, as well as { $breachCount } other breaches.
    }

scan-no-results = 
    Your email address did not appear in our basic scan.
    That's good news, but data breaches can happen any time and there is still more you can do. 
    Subscribe to Firefox Monitor for a full report, alerts when new breaches happen, and tips on protecting your passwords.
    
scan-results = { $breachCount ->
    [one] Your account appeared in the following breach.
  *[other] Your account appeared in the following { $breachCount } breaches.
}
```

Note also that your current string would result in `Your account Your email address did not appear in our basic scan. etc`

Reply:
> I really think we should split out the 0 case. If nothing else for practical reasons: it would be impossible to translate this in Pontoon, where there no text area for variants.

From the Fluent perspective, I think using `[0]` like this is fine and it adds flavor to the translation. The [Good Practices document](https://github.com/projectfluent/fluent/wiki/Good-Practices-for-Developers#prefer-separate-messages-over-variants-for-ui-logic) has more discussion about this.

However, if Pontoon doesn't support extra variants right now, we should avoid it (for now). Sorry, @lesleyjanenorton! We'll get there eventually :)